packages/dapp/src/scripts/graph.js

```diff
@@ -1777,6 +1784,24 @@ function populateSectionEntry(entry, question) {
     entry.attr('data-contract-question-id', contractQuestionID(question));
     //entry.find('.questions__item__title').attr('data-target-id', target_question_id);
 
+    if (question_json['format'] === "markdown"){
+        console.log('yoooo');
+        const unsafeMarkdownHTML = marked.parse(question_json['title']);
```

Comment:
I think we could move this logic to reality-eth-lib: If you've given it the format markdown but it doesn't pass the sanity checks, it should declare the whole question broken and treat it the same way as it would json that doesn't parse or a format that isn't supported.

Reply:
Add this checking to the [reality-eth-lib](https://github.com/shotaronowhere/reality-eth-monorepo-with-markdown/blob/908ec464ce19213412647ce38367598475098b38/packages/reality-eth-lib/formatters/question.js#L233)